js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -287,7 +287,7 @@ var TreeViewBase = HierarchicalCollectionWidget.inherit({
             element: jQuery.fn.init[1]
             itemData: Object
             itemElement: jQuery.fn.init[1]
-            jQueryEvent: jQuery.Event
+            jQueryEvent: eventType
```

Comment:
I think all this comment block should be removed

Reply:
Done.